package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "bluebird": "^3.0.1",
+    "jshint": "^2.9.1",
```

Comment:
Peer dependencies don't work that way, at least not in npm 3.7.3.

```
[bjn@spiral:~/src/umzug]$ rm -rf node_modules
[bjn@spiral:~/src/umzug]$ npm install
...
umzug@1.9.1 /home/bjn/src/umzug
├── bluebird@3.3.4 
├── coffee-script@1.10.0
...
│ │ ├── os-tmpdir@1.0.1 
│ │ └── rimraf@2.2.8 
│ └── through@2.3.8 
├── UNMET PEER DEPENDENCY jshint@2.x
├─┬ jshint-stylish@2.1.0 
│ ├── log-symbols@1.0.2 
│ ├── string-length@1.0.1 
│ └── text-table@0.2.0 
...
  └── yargs-parser@2.1.2 

npm WARN gulp-jshint@2.0.0 requires a peer of jshint@2.x but none was installed.
```

But regarding devDependencies -- true. I'll fix that.